R/copyCohorts.R

```diff
@@ -6,30 +6,81 @@
 #' @inheritParams cohortDoc
 #' @inheritParams nameDoc
 #' @inheritParams cohortIdSubsetDoc
+#' @param n Number of times to duplicate the selected cohorts.
 #'
 #' @return A new cohort table containing cohorts from the original cohort table.
 #' @export
 #'
 #' @examples
 #' library(CohortConstructor)
 #' cdm <- mockCohortConstructor()
-#' cdm$cohort3 <- copyCohorts(cdm$cohort1, name = "cohort3")
-copyCohorts <- function(cohort, name, cohortId = NULL) {
+#' cdm$cohort3 <- copyCohorts(cdm$cohort1, n = 2, cohortId = 1, name = "cohort3")
+#'
+copyCohorts <- function(cohort, n, cohortId = NULL, name = tableName(cohort)) {
```

Comment:
@nmercadeb how about setting the default of n to n = 1 (and then can also move it after cohortId)